tests/TestCase/Database/Type/JsonTypeTest.php

```diff
@@ -133,4 +134,19 @@ public function testToStatement()
     {
         $this->assertSame(PDO::PARAM_STR, $this->type->toStatement('', $this->driver));
     }
+
+    /**
+     * Test instance options
+     *
+     * @return void
+     */
+    public function testInstanceOptions()
+    {
+        // New instance to prevent others tests breaking
+        $instance = new JsonType();
+        $instance->setJsonOptions(JSON_UNESCAPED_UNICODE + JSON_UNESCAPED_SLASHES);
```

Comment:
```suggestion
        $instance->setJsonOptions(JSON_UNESCAPED_UNICODE | JSON_UNESCAPED_SLASHES);
```

Binary flags are usually combined with bitwise OR.

Reply:
Semantically, `+` is more readable for me (adding options to others), BUT, i will change ;)